docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -4,7 +4,7 @@ pagination_prev: getting-started/index
 pagination_next: getting-started/configure-cloud-provider-access/index
 ---
 
-# Install Resoto with Kubernetes
+# Install Resoto with Kubernetes(Helm Chart)
```

Comment:
```suggestion
# Install Resoto with Kubernetes
```

Reply:
I think install with Kubernetes is misleading as there are a lot of methods to install via helm, kubernetes manifest, terraform package. We should be more specific about what we are making users do.

Reply:
It is the same with Docker; there are multiple ways to install. However, we only support one method.

Reply:
Yes, so we should highlight the way we support which is via the helm-chart. Not sure if there is any other way to run docker containers except via the docker agent. A container can be packaged differently to run on top of these orchestration platforms whether be kubernetes, swarm or compone.

Reply:
I disagree. Please revert this change, but we can describe/clarify below that a Helm chart is provided/used.

Reply:
I am still not convinced as `With Kubernetes` sounds vague, we are talking about installing resoto on a Platform via a installation method(helm in this regards).